addons/stock/migrations/9.0.1.1/pre-migration.py

```diff
@@ -28,9 +28,28 @@
 ]
 
 
+def fix_act_window(env):
+    """Action window with XML-ID 'stock.action_procurement_compute' has
+    set src_model='procurement.order', and this will provoke an error as
+    new definition doesn't overwrite this field. We empty that value before
+    that happens. The source of this assignation is not clear, but it doesn't
```

Comment:
You can change this text for adapting according new investigations

Reply:
Done